tensorflow/python/ops/ctc_ops.py

```diff
@@ -47,6 +47,18 @@ def ctc_loss(inputs, labels, sequence_length,
   max(labels.indices(labels.indices[:, 1] == b, 2))
     <= sequence_length(b) for all b.
   ```
+  
+  Useful Tips:
+  
+  The ctc_loss has as input an affine projection of RNN state:
```

Comment:
a more precise description is on ctc_ops.cc
"This class performs the softmax operation for you, so inputs should be e.g. linear projections of outputs by an LSTM."
this also gives the important undocumented info that there is no need to apply softmax before supplying inputs, this is done internally for you